Did not ignore .git direcotry.

```
helm package .git --destination ../output
Error: chart metadata (Chart.yaml) missing
##[error]The process 'helm' failed with exit code 1
##[error]Node run failed with exit code 1
```

If my folder is set to the root directory of project, it will run into `.git` directory and failed.

--- a/index.js
+++ b/index.js
@@ -12,6 +12,7 @@ const getDirectories = fileName =>
     withFileTypes: true,
   })
     .filter(dirent => dirent.isDirectory())
+    .filter(dirent => !(/(^|\/)\.[^\/\.]/g).test(dirent))
     .map(dirent => dirent.name);
 
 async function run() {

--- a/index.js
+++ b/index.js
@@ -60,7 +60,7 @@ async function run() {
     const chartDirectories = getDirectories(path.resolve(`./${chartsDir}`));
 
     console.log('Charts dir content');
-    await exec.exec(`ls`, [], { cwd: `./${chartsDir}` });
+    await exec.exec(`ls`, ['-I ".*"'], { cwd: `./${chartsDir}` });
     for (const chartDirname of chartDirectories) {
       console.log(`Packaging helm chart in directory ${chartDirname}`);
       await exec.exec(

--- a/index.js
+++ b/index.js
@@ -64,7 +64,7 @@ async function run() {
     for (const chartDirname of chartDirectories) {
       console.log(`Resolving helm chart dependency in directory ${chartDirname}`);
       await exec.exec(
-        `helm dependency update && helm dependency build`,
+        `helm dependency update`,
         [],
         { cwd: `./${chartsDir}/${chartDirname}` }
       );

--- a/index.js
+++ b/index.js
@@ -62,6 +62,13 @@ async function run() {
     console.log('Charts dir content');
     await exec.exec(`ls`, ['-I ".*"'], { cwd: `./${chartsDir}` });
     for (const chartDirname of chartDirectories) {
+      console.log(`Resolving helm chart dependency in directory ${chartDirname}`);
+      await exec.exec(
+        `helm dependency update && helm dependency build`,
+        [],
+        { cwd: `./${chartsDir}/${chartDirname}` }
+      );
+      
       console.log(`Packaging helm chart in directory ${chartDirname}`);
       await exec.exec(
         `helm package`,

